FoundryVTTRuleExtension/__init__.py

```diff
@@ -0,0 +1,87 @@
+from PySide6 import QtWidgets, QtCore, QtGui
+from EventBus import EventBus
+from Core.DatenbankEinstellung import DatenbankEinstellung
+from Wolke import Wolke
+import DatenbankEditor
+import logging
+from .DatenbankEditRegelWrapperPlus import DatenbankEditRegelWrapperPlus
+from .RegelExtension import extend_regel
+
+__version__ = "1.0.0"  # Plugin Version
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
```

Comment:
Ich bin recht sicher, dass du hiermit den Sephrasto-logger umkonfigurierst (Loglevel einstellbar in den Einstellungen), die Zeile muss denke ich raus.